src/components/molecules/Questions.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { RadioGroup } from "../molecules/RadioGroup";
+import { PrevNextBar } from "../molecules/PrevNextBar";
+
+export function Questions(props) {
+  const { id, required, textRequired, legend, name, options, onChange } = props;
+
+  function onChangeHandler(e) {
+    onChange(e);
+  }
+
+  return (
+    <div className="max-w-lg">
+      <RadioGroup
+        id={id}
+        required={required}
+        textRequired={textRequired}
+        legend={legend}
+        name={name}
+        options={options}
+        onChange={() => onChangeHandler()}
+      />
+
+      <PrevNextBar
```

Comment:
when prev next is fixed pass handler for when next button or prev button is fixed